api/src/main/java/com/epam/pipeline/manager/datastorage/DataStorageManager.java

```diff
@@ -1205,7 +1204,28 @@ private void assertToolsToMount(final DataStorageVO dataStorageVO) {
         String paramDelimiter = paramValue.contains(",") ? "," : ";";
         for (String path : paramValue.split(paramDelimiter)) {
             if (path.toLowerCase().trim().startsWith(mask.toLowerCase())) {
-                links.add(DataStorageUtils.constructDataStorageLink(dataStorage, path, mask));
```

Comment:
just return state of the code as it was before I extract it to `constructDataStorageLink`